pages/add/form/detail-info-fields.js

```diff
@@ -84,6 +85,16 @@ let Tags = React.createClass({
   }
 });
 
+let IssuesLabel = React.createClass({
+  render() {
+    return(
+      <div>
+        Check any <Link to="/issues">Key Internet Issues</Link> that relate to your project.
+      </div>
+    );
+  }
+});
```

Comment:
Hey @arush0311 sorry that I didn't catch this earlier - I think we should make the link open in another tab (i.e., adding `target="_blank"`) otherwise when someone clicks on the `Key Internet Issues` link he/she will lose all stuff that he/she has already typed in in the form.

Also a note for stateless components you can define them as

```jsx
const IssuesLabel = function() {
  return (
    <div>Check any <Link to="/issues" target="_blank">Key Internet Issues</Link> that relate to your project.</div>
  )
};
```

OR in ES6

```jsx
const IssuesLabel = () => <div>Check any <Link to="/issues" target="_blank">Key Internet Issues</Link> that relate to your project.</div>;
```